Demo/UIKit/Info.plist

```diff
@@ -128,13 +126,17 @@
 		<string>weixinULAPI</string>
 		<string>weixin</string>
 	</array>
+	<key>LSMinimumSystemVersion</key>
+	<string>12.0.0</string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
 	<key>NSAppTransportSecurity</key>
 	<dict>
 		<key>NSAllowsLocalNetworking</key>
 		<true/>
 	</dict>
+	<key>NSCameraUsageDescription</key>
+	<string>Scan credit card details</string>
```

Comment:
This should be in the docs, and this seems to be a breaking change actually. My guess that a simple upgrade to the next SDK version will cause app crash on "scan your card" button tap. Ned to investigate and make a decision if necessary.
@erenbesel @neelSharma12 @nauaros

Reply:
It is a bot of overthinking from my side. It would not crash unless one imports AdyenCardScanner.